src/components/googleMaps/GoogleMaps.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
+    <div class="Google">Google API goes here</div>
```

Comment:
Jeżeli nazwa klasy nawiązuje do nazwy komponentu to powinna być taka sama jak nazwa komponentu, czyli w tym przypadku "GoogleMaps"

Reply:
+1